### PR TITLE
Add low battery documentation

### PR DESCRIPTION
**Pull request by sander76**
Low battery event documentation

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant

https://github.com/home-assistant/home-assistant/pull/22594

### CODE REVIEW

File: source/_components/homematic.markdown [hunk 1]
```diff
@@ -277,6 +278,25 @@ To get the `homematic.keypress` event for some HomeMatic IP devices like WRC2 /
 8. When your channel is working now, you can edit it to select the other channels one by one
 9. At the end, you can delete this program from the CCU
 
+#### {% linkable_title Low battery event %}
+
+As described above devices with buttons are not implemented as actual entities inside Home Assistant. Therefore you cannot watch a low battery state on these.
+To solve this for all homematic IP devices (including those devices with buttons) the event **homematic.low_bat** is fired when battery is running low. you can then use this in your automations:
```
Comment (danielperna84) [inline source/_components/homematic.markdown]: Capital _H_, and actually I think this isn't limited to just IP devices. The regular wireless devices with batteries should behave the same way. 🤔

Comment (sander76) [reply source/_components/homematic.markdown]: Darn. Sloppy coder I am... I can't check on the regular wireless devices. Shall I remove "IP" ?

Comment (danielperna84) [reply source/_components/homematic.markdown]: I'm 99% sure it will work. Can't test right now, but if it doesn't then I'll take care of the fix.

**Review by danielperna84 — APPROVED** [review]
Looking good now. Waiting for tests in Home Assistant to complete.

**Comment by frenck** [comment]
Closing PR, since the parent PR has been closed.

**Pull request closed without merging**